src/components/AddRtspComponent.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { FC } from "react";
+import { useServerSdk } from "./ServerSdkContext";
+import { useAtom } from "jotai";
+import { atomWithStorage } from "jotai/utils";
+
+type Props = {
+  roomId: string;
+  refetchIfNeeded: () => void;
+};
+
+const urlAtom = atomWithStorage("rtsp-url", "");
+const portAtom = atomWithStorage("rtsp-port", "7000");
+const portAutoIncrementAtom = atomWithStorage("rtsp-port-auto-increment", true);
+const keepAliveIntervalAtom = atomWithStorage("rtsp-keep-alive-interval", "");
+const pierceNatAtom = atomWithStorage("rtsp-pierce-nat", true);
+const reconnectDelayAtom = atomWithStorage("rtsp-reconnect-delay", "");
+
+const AddRtspComponent: FC<Props> = ({ roomId, refetchIfNeeded }) => {
+  const { componentApi } = useServerSdk();
+  const [url, setUrl] = useAtom(urlAtom);
+
+  const [port, setPort] = useAtom(portAtom);
+  const parsedPort = parseInt(port);
+  const [autoIncrement, setAutoIncrement] = useAtom(portAutoIncrementAtom);
+
+  const [keepAliveInterval, setKeepAliveInterval] = useAtom(keepAliveIntervalAtom);
+  const parsedKeepAliveInterval = parseInt(keepAliveInterval);
+
+  const [pierceNat, setPierceNat] = useAtom(pierceNatAtom);
+
+  const [reconnectDelay, setReconnectDelay] = useAtom(reconnectDelayAtom);
+  const parsedReconnectDelay = parseInt(keepAliveInterval);
+
+  return (
+    <div className="w-full card bg-base-100 shadow-xl indicator">
+      <div className="card-body p-4">
+        <div className="flex flex-col">
+          <div className="flex flex-col gap-2">
+            <input
+              value={url}
+              onChange={(e) => setUrl(e.target.value)}
+              className="input input-bordered w-full"
+              placeholder="URL"
+            />
+            <div className="flex w-full gap-2">
+              <input
+                value={keepAliveInterval}
+                onChange={(e) => setKeepAliveInterval(e.target.value)}
+                className="input input-bordered flex-1"
+                placeholder="Keep alive interval"
+              />
+              <input
+                value={reconnectDelay}
+                onChange={(e) => setReconnectDelay(e.target.value)}
+                className="input input-bordered flex-1"
+                placeholder="Reconnect delay"
+              />
+              <div className="flex flex-col justify-center tooltip" data-tip="Pierce NAT">
```

Comment:
![image](https://github.com/jellyfish-dev/jellyfish-dashboard/assets/3111202/7dd3a870-4d9f-43f1-b965-61da7e4accf8)
It gets clipped by other elements of the website